packages/jellyfish-api-core/src/index.ts

```diff
@@ -78,10 +78,10 @@ export class ClientApiError extends ApiError {
  * API RPC error, from upstream.
  */
 export class RpcApiError extends ApiError {
-  public readonly payload: { code: number, message: string }
+  public readonly payload: { code: number, message: string, rpcMethod?: string }
 
-  constructor (error: { code: number, message: string }) {
-    super(`RpcApiError: '${error.message}', code: ${error.code}`)
+  constructor (error: { code: number, message: string, rpcMethod?: string }) {
```

Comment:
@fuxingloh  @canonbrother I have added some changes on the `RpcApiError` class to make the rpcmethod visible on error logs.
![image](https://user-images.githubusercontent.com/25013382/118914252-76227980-b95d-11eb-8451-956cc98a9c8b.png)

Tests are running well locally but throwing errors on CI. Let me know if it's ok to merge, otherwise I can remove it when it's ready for review.

Reply:
I think varying CI conditions and CPU differences causes inconsistency in the test. 

Likely because the fees required to send increased hence the insufficient funds. To keep it consistent you can subtract fees from the amount. And with walletBalanceGTE() it will always have sufficient funds. By doing this you might need to increase the amount sent by x10.

https://github.com/DeFiCh/jellyfish/blob/82b6692b48492bf109f95bbe3767f03bf32e0d7a/packages/jellyfish-api-core/src/category/wallet.ts#L259

Reply:
Thanks! will try that